sphinx_gallery/gen_rst.py

```diff
@@ -498,11 +497,9 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     ref_fname = example_file.replace(os.path.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot'):
-        convert_func = dict(code=codestr2rst, text=text2string)
-        for blabel, bcontent in script_blocks:
-            example_rst += convert_func[blabel](bcontent) + '\n'
-    else:
+    filename_pattern = gallery_conf.get('filename_pattern',
+                                        os.path.sep + 'plot')
```

Comment:
I think is enough to have it on the default configuration dictionary in gen_gallery.rst, no need to rewrite in here the default.